packages/lib/pod/src/podTypes.ts

```diff
@@ -68,11 +68,29 @@ export const POD_INT_MIN = 0n;
  */
 export const POD_INT_MAX = (1n << 63n) - 1n;
 
+/**
+ * POD value for EddSA (Baby Jubjub) public keys. Such a value is represented as
+ * a hex string of the (32-byte) compressed form of the key.
+ */
+export type PODEdDSAPublicKeyValue = {
+  type: "eddsa-pubkey";
```

Comment:
Maybe use underscore rather than hyphen?  I feel like it would be safest in future if these types were valid PODNames, and also could turn into variable/class names in other languages.